src/main/java/core/basesyntax/service/impl/ReaderServiceImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReaderService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class ReaderServiceImpl implements ReaderService {
+    @Override
+    public List<String> read(String fileName) {
```

Comment:
```suggestion
    public List<String> read(String fileName) {
    if( fileName == null) {
    throw new RuntimeExeption("Can’t  read file by null path");
    }
```
better use something like this